docs/astro.config.mts

```diff
@@ -9,6 +9,9 @@ import { glob } from 'glob';
 // https://astro.build/config
 const config = defineConfig({
   site: "https://joist-orm.io",
+  redirects: {
+    "/advanced/transform-properties": "/faq/#why-is-joist-transform-properties-no-longer-required",
```

Comment:
This automatically excludes the page in the `robots.txt` file. It does a client-side redirect since it's a staticly built site.